Source: in upstream's cvs repository (for gutenprint-5.2.8)
Upstream: yes, http://sourceforge.net/mailarchive/forum.php?thread_name=AB79C3B9-326A-431A-98AB-7D9D5AB88585%40apple.com&forum_name=gimp-print-devel
Reason: Fixes compilation with cups 1.5

--- a/src/cups/commandtoepson.c	2007/12/23 17:31:51	1.3
+++ b/src/cups/commandtoepson.c	2011/08/13 16:24:17	1.4
@@ -1,5 +1,5 @@
 /*
- * "$Id: commandtoepson.c,v 1.3 2007/12/23 17:31:51 easysw Exp $"
+ * "$Id: commandtoepson.c,v 1.4 2011/08/13 16:24:17 rlk Exp $"
  *
  *   EPSON ESC/P2 command filter for the Common UNIX Printing System.
  *
@@ -32,6 +32,7 @@
 #include <ctype.h>
 #include <stdlib.h>
 #include <string.h>
+#include <stdio.h>
 
 /*
  * Macros...
@@ -229,5 +230,5 @@
 
 
 /*
- * End of "$Id: commandtoepson.c,v 1.3 2007/12/23 17:31:51 easysw Exp $".
+ * End of "$Id: commandtoepson.c,v 1.4 2011/08/13 16:24:17 rlk Exp $".
  */
--- a/src/cups/commandtocanon.c	2001/01/22 20:22:54	1.1
+++ b/src/cups/commandtocanon.c	2011/08/13 16:24:17	1.2
@@ -14,6 +14,7 @@
 #include <cups/cups.h>
 #include <ctype.h>
 #include <stdlib.h>
+#include <stdio.h>
 
 
 /*
